src/test/java/Entities/BoardTest.java

```diff
@@ -0,0 +1,163 @@
+package Entities;
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+import java.util.*;
+
+public class BoardTest {
+    @Test
+    public void testGetPlayers(){
+        Player playerOne = new Player("PlayerOne");
```

Comment:
I think that one thing we can all do after is to refactor our test code so that we use the @before annotation to specify the setting up for the tests so that we don't need to have redundant code in every test case. But other than that this looks good.